src/Core/Grid/Query/Filter/DoctrineFilterApplicator.php

```diff
@@ -80,8 +85,67 @@ public function apply(QueryBuilder $qb, SqlFilters $filters, array $filterValues
                         $qb->setParameter($name, sprintf('%s %s', $value['to'], '23:59:59'));
                     }
 
+                    break;
+                case SqlFilters::MIN_MAX:
```

Comment:
Add unit tests for this modification.

Reply:
Agreed ✅  this is a good candidate for unit testing

Reply:
Added the unit test